app/src/main/scala/org/alephium/explorer/web/UtilsServer.scala

```diff
@@ -39,5 +47,58 @@ class UtilsServer()(implicit ec: ExecutionContext,
     toRoute(sanityCheck) { _ =>
       sideEffect(SanityChecker.check())
       Future.successful(Right(()))
+    } ~
+      toRoute(changeGlobalLogLevel) { level =>
+        Future.successful(updateGlobalLevel(level))
+      } ~
+      toRoute(changeLogConfig) { logbackConfig =>
+        Future.successful(updateLoggerContext(logbackConfig))
+      }
+
+  @SuppressWarnings(Array("org.wartremover.warts.AsInstanceOf"))
+  private def updateGlobalLevel(level: String): Either[ApiError[_ <: StatusCode], Unit] = {
+    val root = LoggerFactory.getLogger(org.slf4j.Logger.ROOT_LOGGER_NAME).asInstanceOf[Logger];
+    Try(root.setLevel(Level.valueOf(level))) match {
+      case Success(_) =>
+        logger.info(s"Logging level updated to $level.")
+        Right(())
+      case Failure(error) =>
+        val errorMessage = s"Cannot apply logback configuation: $error"
+        logger.error(errorMessage)
+        Left(ApiError.BadRequest(errorMessage))
+    }
+  }
+
+  private def updateLoggerContext(
+      logbackConfig: String): Either[ApiError[_ <: StatusCode], Unit] = {
+
+    val loggerFactory = LoggerFactory.getILoggerFactory()
+
+    loggerFactory match {
+      case logbackCtx: LoggerContext =>
+        val configurator = new JoranConfigurator();
+        logbackCtx.reset
+        configurator.setContext(logbackCtx);
+
+        val stream = new java.io.ByteArrayInputStream(
+          logbackConfig.trim().getBytes(java.nio.charset.StandardCharsets.UTF_8.name)
+        )
+
+        Try(configurator.doConfigure(stream)) match {
+          case Success(_) =>
+            logger.info("Logback configuration updated OK.")
+            Right(())
+          case Failure(e) =>
+            val init = new ContextInitializer(logbackCtx)
+            init.autoConfig
+            logger.warn(
+              s"Cannot apply logback configuation, Logback configuration reset to what is defined on the class path or system properties: ${e.getMessage}")
+            Left(ApiError.BadRequest(s"Cannot apply logback configuation: ${e.getMessage}"))
```

Comment:
Being a user facing API we might as well fix typos: `configuration` :) Also at 2 other places in this file.

Reply:
done